### DISCUSSION TITLE
Incorrect stats in the card

### DISCUSSION THREAD

**Discussion by rahul-vashishtha** [anuraghazra/github-readme-stats#3385] [Q&A] [answered]
### Describe the bug

The card has recently been showing incorrect stats.
In the card, it's only showing that I have only 25 commits in 2023 with rating B-, which is wrong. I have more than 1000 commits in 2023 itself.

Here is my Github Profile Link: https://github.com/rahul-vashishtha

**Comment by qwerty541** [comment, marked as answer]
Hey, @rahul-vashishtha! Yes, i see that you have a lot of contributions, but most of them are private. Our documentation says that by default the stats card shows statistics only from public repositories. To show private statistics on the stats card, you should [deploy your own instance](https://github.com/anuraghazra/github-readme-stats#deploy-on-your-own) using your own GitHub API token.

![image](https://github.com/anuraghazra/github-readme-stats/assets/53787217/d2745ae3-8a5b-47f5-b776-9d0594c2e60f)

**Reply by rahul-vashishtha** [reply]
Hey @qwerty541,
Yes, the instance is deployed on my own Vercel. Still its not showing data. It was showing previously, but suddenly stopped. And GitHub token is not expired too.

**Reply by qwerty541** [reply]
Did you set correct permissions for your PAT (Personal Access Token (GitHub API key))? Make sure that you enabled `repo` and `user` permissions.

![image](https://github.com/anuraghazra/github-readme-stats/assets/53787217/9c94bb06-a59d-402e-8014-5f821046b630)

![image](https://github.com/anuraghazra/github-readme-stats/assets/53787217/c835b969-9850-4cc9-82ed-b9207214b47f)
